bitnami/nginx-ingress-controller/values-production.yaml

```diff
@@ -260,33 +329,15 @@ extraInitContainers: []
 #   image: bitnami/minideb
 #   command: ['sh', '-c', 'install_packages dnsutils && until nslookup myservice; do echo waiting for myservice; sleep 2; done;']
 
-stats:
-  enabled: true
-
-  service:
-    annotations: {}
-    clusterIP: ""
-
-    ## List of IP addresses at which the stats service is available
-    ## Ref: https://kubernetes.io/docs/user-guide/services/#external-ips
-    ##
-    externalIPs: []
-
-    loadBalancerIP: ""
-    loadBalancerSourceRanges: []
-    port: 18080
-    type: ClusterIP
-
-## If stats.enabled = true and metrics.enabled = true, Prometheus metrics will be exported
-##
 metrics:
   enabled: true
 
   service:
-    annotations:
-      prometheus.io/scrape: "true"
-      prometheus.io/port: "10254"
+    annotations: {}
+    # prometheus.io/scrape: "true"
```

Comment:
If we are enabling the metrics by default, then we should uncomment these values

Reply:
Otherwise the metrics would not work